### ISSUE TITLE
NOT AN ISSUE: OpenCV or FFmpeg?

### ISSUE THREAD

**Issue by baqwas** [GlassOnTin/picamera2-webstream#2]
For a dedicated and basic video stream, which makes more sense - OpenCV or FFmpeg?

Also, you mention virtual environment, but most of the requisite packages seem to be installed outside the virtual environment in your script. Is my understanding correct?

**Comment by GlassOnTin** [comment]
**FFMPEG vs Picamera2 / OpenCV**
For basic video streaming, yes, I'd agree FFmpeg is better suited than OpenCV.
FFmpeg has lower overhead for pure streaming.
OpenCV adds unnecessary complexity when just streaming video.  
But, the picamera2 OpenCV stream has possibilities :)
=> So, I've now created an FFmpeg version (0.2.#) for direct comparison

**Regarding virtual environment usage:**
Yes - the current script installs some dependencies system-wide.  
This is because for most users of a raspberry pi, picamera2 and its dependencies work best when installed via `apt`.  The split installation approach ensures compatibility with Raspberry Pi OS while keeping application-specific dependencies isolated.
=> I've now made that clear in the README.md

Thanks for the feedback :)
